SparkPost/2web2ui
# FE-1287 - UX feedback for Pinned Report

FE-1287 - UX feedback for Pinned Report 

### What Changed
 
- Added a `small` prop for chart components so that height if a single chart is 200 on dashboard for pinned report
- Radio button are now the part of Name columns in Change Report Modal
- Made the Change Report Modal accessible by keyboard (I can  give a walkthrough for the change here) 
- Saved Report Modal on Analytics Report - Action is always visible to Schedule and Pin, hide Edit and Delete based on whether user can edit or not

### How To Test

- On Dashboard, when the pinned report has single chart, max height should be more than 200.
- On Dashboard, Radio buttons in Change Report Modal are now part of Name Column
- On Dashboard, Radio buttons on Change Report Modal are accessible by keyboard
- On Analytics Report Page, Saved Report Modal - Action is now always visible with Schedule and Pin options, and Edit and Delete are hidden based on whether user can edit or not

### To Do
